frontend/src/components/SystemsTable/SystemTableContent.tsx

```diff
@@ -220,6 +246,37 @@ export function SystemTableContent({
     },
   };
 
+  const handleTableChange = (
+    _pagination: TablePaginationConfig,
+    filters: Record<string, FilterValue | null>,
+    _sorter: SorterResult<SystemModel> | SorterResult<SystemModel>[]
+  ) => {
+    console.log(filters);
+    for (const k in filters) {
```

Comment:
Thank you very much! This looks much clearer. I add another boolean flag before `onFilterChange` because `onFilterChange` will set the pagination to page one even when `filterUpdate` is empty.